### DISCUSSION TITLE
[FEATURE] <enhancements>

### DISCUSSION THREAD

**Discussion by andsofine** [libre-tube/LibreTube#465] [General]
• high quality covers and other pics.
• thumbnail preview on the timeline
• embedded timecodes
• pull down to refresh
• share link with timecode
• show password when logging in
• search filters
• 4x speed
• skip silence
• volume and brightness control by swipe (option to enable in settings)
• comments + comments translation (with the ability to choose a service for translation as in Infiniti for reddit)
• library (the ability to remove videos from playlists (history, personal playlists) using swipe)
• the ability to expand the thumbnailed video with a tap.
• shorts (maybe disabled by default but can be enabled in settings)

idea from myself. maybe not the most urgent or obligatory thing, but I think it would be very convenient

a separate tab with live streams of my subscriptions. or the ability to enable such a separate tab in the settings. in this case, live streams will be excluded from the subscription list.

then you can still create (or enable) a separate recommendation tab where only live streams are displayed.
you can also do the same for recommendations (you can also enable it in the settings). two sub-tabs in the recommendations. in one subtab only videos in the other only live streams.

sorry for my english

**Comment by Silther** [comment]
Can this be closed as almost all features are now implemented?
